src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -115,7 +115,8 @@ public Task<string> GetFileContentsAsync(string filePath, string repoUri, string
                         $"_apis/git/repositories/{repoName}/items?path={filePath}&versionType={versionType}&version={branchOrCommit}&includeContent=true",
                         _logger,
                         // Don't log the failure so users don't get confused by 404 messages popping up in expected circumstances.
-                        logFailure: false);
+                        logFailure: false,
+                        retryCount: 0);
```

Comment:
>retryCount: 0 [](start = 24, length = 13)

zero?  Is it ok for this operation to fail? Azure DevOps definitely fails some requests (though the default of 15 is pretty extensive).  Especially given how many requests we make before this is all said and done, I'd expect the aggregated failure rate to get pretty high.

Reply:
Yes, this is intentional.  AzDO does not provide APIs that take a 'commitish' (tag/branch/sha).  You have to know what you're talking about.  So in this case we have to actually make a call and see whether we 404 on calling something a commit or a branch or whatnot to determine what it is.

Obviously this will fail intentionally.  I've not yet seen it actually fail when it's not supposed to, so a retry of 0 is intended.